code/modules/atmospherics/he_pipes.dm

```diff
@@ -100,6 +103,10 @@
 				for (var/thing in animate_targets)
 					var/atom/movable/AM = thing
 					animate(AM, color = rgb(h_r, h_g, h_b), time = 20, easing = SINE_EASING)
+				filters[1].color = rgb(h_r, h_g, h_b)
```

Comment:
Not sure I like this. Can you store the `filter()` return somewhere instead?

Reply:
This also isn't updating the color of mimics.

Reply:
The mimics don't have any filters - the filters are on the base H/E pipe itself, so it doesn't update the mimic filters as intended.